### DISCUSSION TITLE
Change sub languege to Spanish

### DISCUSSION THREAD

**Discussion by neyberson** [pystardust/ani-cli#1156] [General] [answered]
Hi, I was using this awesome tool, but I want to know if I can change the sub language to Spanish

**Comment by justchokingaround** [comment, marked as answer]
no you can't, since the script scrapes allanime.to, which does not have an option for subtitles

**Reply by jorgegso** [reply]
hi, It is possible to put it in English and subtitle it in English as well?

**Reply by justchokingaround** [reply]
use `--dub`
